pyproject.toml

```diff
@@ -141,6 +141,25 @@ match-dir = '^(?!docs|tutorials|build|dist|\.).*'
 add_ignore = 'D100,D104,D107'
 
 [tool.pytest.ini_options]
+minversion = '6.0'
+addopts = '--durations 20 --junit-xml=junit-results.xml --verbose'
 filterwarnings = [
     "ignore:Matplotlib is currently using agg:UserWarning",
 ]
+
+[tool.coverage.run]
+branch = true
+cover_pylib = false
+omit = [
+    '**/__init__.py',
+    '**/pycrostates/_version.py',
+    '**/pycrostates/utils/_fixes.py',
+    '**/tests/**',
+]
+
+[tool.coverage.report]
+exclude_lines = [
+    'pragma: no cover',
+    'if __name__ == .__main__.:',
+]
+precision = 2
```

Comment:
This setup reduces the code coverage percentage, but now it reflects more the reality. Especially removing all the `tests` files which by definition are covered at 100%.